sentry/src/main/java/io/sentry/SentryMeasurementUnit.java

```diff
@@ -0,0 +1,37 @@
+package io.sentry;
+
+import java.util.Locale;
+import org.jetbrains.annotations.NotNull;
+
+public enum SentryMeasurementUnit {
```

Comment:
Should we add a comment with link to the docs here?

```suggestion
/**
 * @see <a href="https://docs.sentry.io/platforms/python/guides/pyramid/performance/instrumentation/performance-metrics/">Develop Docs</a> 
 */
public enum SentryMeasurementUnit {
```

Not sure if there's user facing docs available.

Reply:
I think I wouldn't want to link to python docs here :D changed it to Relay docs https://getsentry.github.io/relay/relay_metrics/enum.DurationUnit.html

Also I just noticed we only offer DurationUnit here actually (and also in the dart sdk cc @marandaneto), should we actually provide all of them? Or at least rename this one to something like `SentryMeasurementUnit.Duration` or so https://getsentry.github.io/relay/relay_metrics/enum.MetricUnit.html

Reply:
Yeah we can add more units. Oopsie on the docs link. Just hit search and didn't notice where it took me^^